drivers/storage/portworx/component/telemetry.go

```diff
@@ -116,7 +116,6 @@ type telemetry struct {
 	isDeploymentRegistrationServiceCreated bool
 	isDaemonSetTelemetryPhonehonmeCreated  bool
 	usePxProxy                             bool
-	reconcileMetricsCollector              *bool
```

Comment:
you may want to keep the code but set it to true just in case there is still issue in future